subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -340,7 +340,7 @@ uint8_t ll_adv_params_set(uint16_t interval, uint8_t adv_type,
 	adv->lll.chan_map = chan_map;
 	adv->lll.filter_policy = filter_policy;
 
-#if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY)
+#if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY) && defined(CONFIG_BT_CTLR_ADV_EXT)
```

Comment:
Why `CONFIG_BT_CTLR_ADV_EXT` ?

Reply:
It's because I'm setting `CONFIG_BT_CTLR_SCAN_REQ_NOTIFY=y` independently of `CONFIG_BT_CTLR_ADV_EXT`, as it seems possible. (Before I did it in the application, now it's selected by `BT_CTLR_VS_SCAN_REQ_RX`).